### ISSUE TITLE
Investigate CMS

### ISSUE THREAD

**Issue by lautaro450** [coderbunker/coderbunker.com#43]
## Problem ##
Blog, new events, new members potentially, can all be added through the website. Any non technical user should be able to add content to the site. CMS doesn't have to be a framework, could be a process.

## Possible solutions ##
 - Use a template (google doc or markdown) and wysiwyig for non-technical users to create content. Template for blog, article, workshop, etc...
 - Jekyls

## What's important for the CMS ##
- [ ] look at the DLG project  spreadsheet for costing to add a django cms to a project
  - can also talk with  @lenzai and @Aesirry 
- [ ] localization
- [ ] article/blogging support
- [ ] no coding required for content generation.
- [ ] what does it cost to implement the following templates in each CMS solution
  - workshop, landing page, blog post etc...
- [ ] understand he workflow for using the CMS for someone non-technical (i.e. what do they have to do to make a new workshop
- [ ] come up with a number of hours for Dante to implement
- [ ] cost for infrastructure (hosting, file dump, etc...)

**Comment by echolabstech** [comment]
@TomasJ1993 so another effort to be done in parallel with development is understanding the needs of the project. We need a CMS but we also need to estimate its cost and add that to the cost of the project. We make engineering decisions not just about technology, but also about where to spend our time. Is it cost effective to build in a CMS. We need numbers that we can show to management (@lenzai @rngadam ).

**Comment by echolabstech** [comment]
Meet with @lautaro450 @Sunying-RONG decided CMS is expensive and to use Jekylls to build new pages and add new content upon request.
